web/modules/custom/dpl_admin/dpl_admin.install

```diff
@@ -32,3 +32,18 @@ function dpl_admin_requirements(string $phase): array {
 
   return $requirements;
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * This update ensures that all sites are returned to normal state
+ * (not in maintenance mode) after limiting access to maintenance mode
+ * to only administrators.
+ *
+ * See https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1171
+ */
+function dpl_admin_update_8001(): void {
+  // Set all sites to normal mode.
+  \Drupal::state()->set('system.maintenance_mode', FALSE);
+  \Drupal::messenger()->addMessage(t('All sites have been moved back to normal state.'));
+}
```

Comment:
A few things to consider:

We have already added 2 update hooks 10000 and 10001. I think this new update hook should be `dpl_admin_update_10002`. 

I have experienced that `\Drupal::messenger()` does not always print the message. Instead we are returning a string with the output instead. Consider doing the same here. A sidenote: We don't need to translate the outputted string, so you can safely omit the t() function from this. 

Otherwise this looks good, nice job 👍

Reply:
Thanks 🙏 I have rebased from develop to fix merge conflicts. 

I have also added to `config_perms` in web/modules/custom/dpl_update/dpl_update.install. I chose 1003 because I am installing another module at 1002 in this pull request: https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1241/files